backend/corpora/common/entities/collection.py

```diff
@@ -254,19 +326,19 @@ def tombstone_collection(self):
             ds.tombstone_dataset_and_delete_child_objects()
         self.session.commit()
 
-    def update(self, links: list = None, **kwargs) -> None:
+    def update(self, links: list = None, keep_links=None, **kwargs) -> None:
```

Comment:
Should `keep_links` have a default of `False` instead of none?
Also need to add the `keep_links` param in the docstring below.

Reply:
I blame this on late-night coding a month ago.